README.md

```diff
@@ -12,6 +12,7 @@ Options
 * `channel`  - The Slack channel to override the default channel. (without #).
 * `username` - The name of your bot. (default `Wercker`)
 * `branches` - Specific branches to notify. (regular expression)
+* `notify_on` - Allows you to specify whether to notify on failures only
```

Comment:
I understand that `notify_on` is `passed` or `failed` only.
I think it is better to explain about valid values :bulb: 

e.g.

```
* `notify_on` - Allows you to specify whether to notify on `passed` or `failed`.